src/config/mod.rs

```diff
@@ -408,6 +409,16 @@ impl Config {
         self.engine.replace(engine);
     }
 
+    pub fn get_tracemap_settings(&self) -> TraceMapSettings {
+        TraceMapSettings {
```

Comment:
Oh thoughts on other options that would impact the coverage calculation in the Config struct: `run_types` and `varargs` as including/excluding ignored tests or applying filtering. Technically only some of the varargs would impact the end result but I don't think trying to stay inline with test executable args is smart/useful.